presets/4.3/other/Tehllama_GTB339_3in_SWPR-Spec_2S-3S_Race_TUNE_OTHER.txt

```diff
@@ -0,0 +1,209 @@
+#$ TITLE: 3in Lightweight 3S/2S Tune for GTB-339 or 3" Racing Toothpick 
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: OTHER
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: GTB339, Toothpick, Spec, Race, Llama, SWPR, Multi, Voltage, 2S, 3S
+#$ AUTHOR: Daniel Appel / Tehllama
+#$ DESCRIPTION: This tune is intended for GTB339 3in 1105-5500KV Racers on on bidir DShot ESCs at 48kHz PWM with MedHigh timing. 
+#$ DESCRIPTION: These fly best on 2S-3S batteries in the 450-650mAh capacity range, resulting in an AUW to 95-105g on these motors.  
+#$ DESCRIPTION: This tune works with biblade and triblade props, such as Gemfan 3016-3, GF 3018-2, HQ 3x1.8x3, and HQ 3x1.5x3.
+#$ DESCRIPTION: This tune also works well on other lightweight 1303/1303.5/1204/1106 3" setups in the 4500-6000KV range.
+#$ DESCRIPTION: Extensive testing has been done across 39+ builds, however not every craft will fly well on this tune.
+#$ DESCRIPTION: Strongly recommend a full chip erase reflash if using alternative tune presets is desired.  
+#$ DESCRIPTION: This tune will auto-select profiles based on battery voltage at plugin for 2S and 3S packs with default selections. 
+#$ DESCRIPTION: This preset assumes 12-pole motors by default.  In the unlikely case your motors actually have 14 MAGNETS, de-select this option.
+#$ WARNING: Use at your own risk.  Some 13A ESCs may experience failures on 3S batteries.
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/239
+
+# FORCE_OPTIONS_REVIEW: TRUE
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+
+#$ OPTION_GROUP BEGIN: Tune Profiles
+
+#$ OPTION BEGIN (CHECKED): Apply 3S Auto-Select Tune on Profile 3
+# master
+set dshot_bidir = ON
```

Comment:
setting dshot bidir and other things inside "Apply 3S Auto-Select Tune on Profile 3" option

Reply:
Thank you for the review in detail... that was definitely not intended